spec/features/projects_spec.rb

```diff
@@ -45,6 +45,28 @@
           expect(user.projects.count).to eq(0)
         end
       end
+
+      describe 'archivement' do
+        before { visit edit_project_path(project) }
+
+        context 'with a non archived project' do
+          let(:project) { create :project, users: [user], teams: [current_team] }
+
+          scenario 'there is no button' do
+            expect(page).not_to have_selector(:link_or_button, 'Archive')
```

Comment:
Since the project isn't enabled, why the archive button isn't visible?

Reply:
When the project is not archived, there is no Archive button, but there is an Unarchive button